src/__tests__/unit/lib/supercomputer.test.ts

```diff
@@ -306,6 +306,123 @@ describe('lib/supercomputer: ', () => {
       );
     });
 
+    it('passes output from earlier executed model to the later ones as an input', async () => {
```

Comment:
IMO having all the test related entities and logic inside of the test itself increases readability and maintainability.